src/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+      crossorigin="anonymous"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <title>Javascript Group Capstone Project By David Yannick and Saningo  </title>
+</head>
+<body>
+  <header>
+    <div class="navbar">
+      <a class="link-logo" href="./index.html">Logo</a>
+      <nav class="links">
+        <ul class="link-lists">
+          <li class="link-list"><a href="#" >About</a></li>
+          <li class="link-list"><a href="#" >Contact</a></li>
+          <li class="link-list"><a href="./index.html" class="link tv-show-link">.</a></li>
+        </ul>
+      </nav>
+    </div>
+  </header>
+  <main class="main-content">
+    <div class="grid-cards-container"></div>
+  </main>
+  <aside class="movie-popup"></aside>
+  <footer>
+    Javascript Group Capstone By David Yannick and Saningo
+  </footer>
```

Comment:
- [ ] You have not implemented a requested change by the previous reviewer. She mentioned making your footer similar to the given one. Kindly you can take a look at the screenshot of the original footer design below  and let's fix it 👇 
![Screenshot (256)](https://user-images.githubusercontent.com/67087664/164896172-e843935d-63d4-4ce1-96c7-57b9f28dc25a.png)